sdv/metadata/multi_table.py

```diff
@@ -783,8 +800,12 @@ def validate(self):
         """
         errors = []
         self._validate_single_table(errors)
+        self._seen_foreign_keys = {}
```

Comment:
can we just make this a variable and not store it on the class

Reply:
Done in [81a46ae](https://github.com/sdv-dev/SDV/pull/2477/commits/81a46ae0f4abdf8a8dfbb4eb23db068b6b56fc1a)